docs/guide/deployment/authorization-server.md

```diff
@@ -1,65 +1,158 @@
+| `api.http.tls.caPool` | string | `file path to the root certificates in PEM format` |  `""` |
+| `api.http.tls.keyFile` | string | `file name of private key in PEM format` | `""` |
+| `api.http.tls.certFile` | string | `file name of certificate in PEM format` | `""` |
+| `api.http.tls.clientCertificateRequired` | bool | `require client certificate` | `true` |
+
+### Authorization Client for Device OAuth Provider
+
+| Property | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `oauthClients.device.provider` | string | `value which comes from the device during the sign-up ("apn")` | `"generic"` |
+| `oauthClients.device.clientID` | string | `client id for authentication to get access token/authorization code` | `""` |
+| `oauthClients.device.clientSecret` | string | `client secret for authentication to get access token` |  `""` |
+| `oauthClients.device.scopes` | string | `Comma separated list of required scopes` | `""` |
+| `oauthClients.device.authorizationURL` | string | `authorization endpoint` | `""` |
+| `oauthClients.device.tokenURL` | string | `token endpoint` | `""` |
+| `oauthClients.device.audience` | string | `audience of oauth provider` | `""` |
+| `oauthClients.device.redirectURL` | string | `redirect url used to obtain device access token` | `""` |
+| `oauthClients.device.responseType` | string | `one of "code/token"` | `"code"` |
+| `oauthClients.device.responseMode` | string | `one of "query/post_form"` | `"post_form"` |
+| `oauthClients.device.http.maxIdleConns` | int | `controls the maximum number of idle (keep-alive) connections across all hosts. Zero means no limit.` | `16` |
+| `oauthClients.device.http.maxConnsPerHost` | int | `optionally limits the total number of connections per host, including connections in the dialing, active, and idle states. On limit violation, dials will block. Zero means no limit.` | `32` |
+| `oauthClients.device.http.maxIdleConnsPerHost` | int | `if non-zero, controls the maximum idle (keep-alive) connections to keep per-host. If zero, DefaultMaxIdleConnsPerHost is used.` | `16` |
+| `oauthClients.device.http.idleConnTimeout` | string | `the maximum amount of time an idle (keep-alive) connection will remain idle before closing itself. Zero means no limit.` | `30s` |
+| `oauthClients.device.http.timeout` | string | `a time limit for requests made by this Client. A Timeout of zero means no timeout.` | `10s` |
+| `oauthClients.device.http.tls.caPool` | string | `file path to the root certificates in PEM format` |  `""` |
+| `oauthClients.device.http.tls.keyFile` | string | `file name of private key in PEM format` | `""` |
+| `oauthClients.device.http.tls.certFile` | string | `file name of certificate in PEM format` | `""` |
+| `oauthClients.device.http.tls.useSystemCAPool` | bool | `use system certification pool` | `false` |
+
+### Authorization Client for Service OAuth Provider
+
```

Comment:
Add a note with explanation what is it used for:
`> Configured OAuth2.0 client is used by the mobile application or SDK to request a token used to authorize all calls they execute against the plgd API Gateways.`